lib/rules/component-tags-order.js

```diff
@@ -70,11 +67,74 @@ module.exports = {
     })
 
     /**
-     * @param {string} name
+     * @param {VElement} element
+     * @return {String}
+     */
+    function getAttributeString(element) {
+      return element.startTag.attributes
+        .map((attribute) => {
+          if (attribute.value && attribute.value.type !== 'VLiteral') {
+            return ''
+          }
+
+          return `${attribute.key.name}${
+            attribute.value && attribute.value.value
+              ? '=' + attribute.value.value
+              : ''
+          }`
+        })
+        .join(' ')
+    }
+
+    /**
+     * @param {String} ordering
+     * @param {VElement} element
+     * @return {Boolean} true if the element matches the selector, false otherwise
+     */
+    function matches(ordering, element) {
+      let hasMatch = true
+      let isNegated = false
+
+      parser((selectors) => {
+        selectors.walk((selector) => {
+          switch (selector.type) {
+            case 'tag':
```

Comment:
This is in no way a correct implementation of css selectors, but I think it supports all the features we'll need to get this off the ground.

Reply:
I was wrong, this is busted. I'll get back at it soon; sorry for pushing the re-review button early